website/public/main.css

```diff
@@ -166,6 +166,18 @@ nav.breadcrumbs ol > li:not(:first-child)::before {
   content: "➤ ";
 }
 
+/* make "#" selected sections stand out */
+h4:target::before {
+  content: "👉"; /*TODO(😎): localize for different layout directions (e.g. rtl)*/
+}
+h4:target {
+  transition-property: box-shadow, transform;
+  transition-duration: 333ms;
+  transition-timing-function: ease;
```

Comment:
Minor: If we have an animation, we should respect [prefers-reduced-motion](https://developer.mozilla.org/en-US/docs/Web/CSS/@media/prefers-reduced-motion).